docs/navigating-compliance-json.md

```diff
@@ -0,0 +1,87 @@
+# Navigating Compliance Using Structured Data
+
+> Warning:
+This feature is in early development and is only implemented within the API.  There are multiple caveats that need to be considered and are listed below.
+
+## Caveats
+- Must have a `Platform` created specifically for the feature.
+- Both the compliance `feature` and `rule` need to be unique for the JSON `config-type`.
+- When creating `Compliance Rules` with the config type of JSON, the `config to match` field is not used but must have bogus data in order to save to the DB.
```

Comment:
Let's change this to not required on model and validate when cli, it is required.

Reply:
changed `blank=True`.  Will add further checks.